jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/content/ContentSourceTransformer.java

```diff
@@ -157,4 +159,28 @@ private Content.Chunk process(Content.Chunk rawChunk)
      * @return a transformed chunk or {@code null}
      */
     protected abstract Content.Chunk transform(Content.Chunk inputChunk);
+
+    private class DemandInvocableTask implements Invocable.Task
```

Comment:
Just `DemandTask` and extends `ReadyTask`.

Reply:
I prefer as is, but with final field for InvocationType